.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+*               @Datadog/libdatadog-core
+ddprof*/        @Datadog/profiling
```

Comment:
only library people are relevant, though do you want to create a new group or just list the people here ?

Reply:
Good point. I've added new team libdatadog-profiling and added you all as maintainers.